scripts/build-json/build-page-json.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const bcd = require('./resolve-bcd');
+const examples = require('./compose-examples');
+const attributes = require('./compose-attributes');
+const prose = require('./slice-prose');
+const contributors = require('./resolve-contributors');
+
+const writeToFile = (propertyName, json) => {
+  const data = {
+    html: {
+      elements: {
+        [propertyName]: json,
+      }
+    }
+  };
+
+  const dest = path.join(process.cwd(),'packaged/html/elements', `${propertyName}.json`);
+  const destDir = path.dirname(dest);
+  if (!fs.existsSync(destDir)) {
+      fs.mkdirSync(destDir, { recursive: true });
+  }
+  fs.writeFileSync(dest, `${JSON.stringify(data, null, 2)}`);
```

Comment:
```suggestion
  fs.writeFileSync(dest, JSON.stringify(data, null, 2));
```

Reply:
I don't know Node enough to appreciate the risks of NOT setting the encoding when writing a file. In Python it's utf-8 by default.